pkg/grpc/manager/manager.go

```diff
@@ -153,6 +154,13 @@ func (g *Manager) createLocalConnection(parentCtx context.Context, port int, ena
 	} else {
 		opts = append(opts, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	}
+	opts = append(opts, grpc.WithConnectParams(grpc.ConnectParams{
+		Backoff: backoff.Config{
+			MaxDelay:  5 * time.Second,
```

Comment:
Question: Where are these numbers selected from?

Reply:
Please also add a test for what this is fixing.

Reply:
These numbers came from my intuition :) We can eventually have entries in the Configuration CRD to allow this to be modified.

I am relying on your integration test to validate this, in addition to the Java SDK ITs. This is not unit-testable.